app/services/json_to_events.rb

```diff
@@ -19,16 +19,35 @@ def call
       campus = Campus.where(name: event_hash["campus"]).first_or_create!
 
       # Create an event
-      Event.create!(
+      # Event.create!(
```

Comment:
We can remove the commented out code, since it will be in our Git history :+1:

Reply:
Will do